kclvm/ast_pretty/src/tests.rs

```diff
@@ -46,7 +46,8 @@ fn read_data(data_name: &str) -> (String, String) {
 fn test_ast_printer() {
     for case in TEST_CASES {
         let (data_input, data_output) = read_data(case);
-
+        #[cfg(target_os = "windows")]
+        let data_input = data_input.replace("\n", "\r\n");
```

Comment:
Maybe the `NEW_LINE` definition of AST printer should be modified.

Reply:
I think KCL, like Rust, should have used the LF ending no matter what platform it runs on. CRLF is an anachronism that needs to die.